src/Compilers/CSharp/Test/Syntax/Parsing/ParserErrorMessageTests.cs

```diff
@@ -3505,7 +3458,7 @@ public class TestType
 
         [Fact]
         [CompilerTrait(CompilerFeature.ReadOnlyReferences)]
-        public void BadInWithThisParameterModifiers()
+        public void InWithThis_ParameterModifiers()
```

Comment:
We don't need this test now. VerifyDiagnostics is called on the same combination of modifiers elsewhere.
But if we do want to keep this, it should be moved to another file.